src/number.ts

```diff
@@ -43,14 +43,20 @@ export default class NumberSchema<
     return typeof value === 'number' && !isNaN(value);
   }
 
-  min(min: number | Reference<number>, message = locale.min) {
+  min(min: number | Reference<number>, message = locale.min) {    
     return this.test({
       message,
       name: 'min',
       exclusive: true,
       params: { min },
       test(value: Maybe<number>) {
-        return isAbsent(value) || value >= this.resolve(min);
+        const resolvedMin = this.resolve(min);
+
+        // NOTE: ES "Strict Equality Comparison" (i.e ===) does not differentiate b/w -0 and 0
+        // Ref: https://262.ecma-international.org/6.0/#sec-strict-equality-comparison
+        // "Object.is" implements "Same Value" algo which specifically considers -0 and 0
+        // Ref: https://tc39.es/ecma262/#sec-samevalue
```

Comment:
Would it be possible to avoid abbreviations in your code? So please write `between` instead of `b/w`, and write `algorithm` instead of `algo`.